wire-ios-request-strategy/Sources/Payloads/Payload.swift

```diff
@@ -47,11 +47,35 @@ enum Payload {
         var qualifiedIDs: [QualifiedID]
     }
 
+    struct UserProfilesList: Codable {
```

Comment:
What do you think of naming this `UserProfilesV4` ? Since we already have a `Payload.UserProfiles`, I think that naming this `UserProfilesList` might be a bit confusing. Whereas if we explicitly tie the name to the api version it is meant to support, it would be easier to understand what the purpose of the payload struct is

Reply:
Done